rabbitmq-operator-model/build.gradle

```diff
@@ -0,0 +1,29 @@
+plugins {
+    id 'java'
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
```

Comment:
i believe a whole bunch of these aren't needed

Reply:
I thought I'd verified that all of them were necessary, but it appears that I can remove jackson-databind and the kubernetes-client annotation processor.

I'll check again after moving some of the files out of the model module.